package.json

```diff
@@ -101,7 +101,7 @@
   },
   "dependencies": {
     "react-dom": "^18.2.0",
-    "react-native-permissions": "*",
+    "react-native-permissions": "3.8.4",
```

Comment:
I did lock this version to 3.8.4 to avoid using unstable versions of this package. With the '*' specification I was using the 4.0.2 unreleased version, and this caused problems with the pods on mac and runtime weird exceptions when requesting permissions on Android.